File: 9041/lab05/echon.pl
#!/usr/bin/perl -w

if (@ARGV != 2) {
   print "Usage: ./echon.pl <number of lines> <string>\n";
   exit 0;
}
if ($ARGV[0] =~ /^\d+$/) {
	foreach (1..$ARGV[0]) {
    		print "$ARGV[1]\n"
	} 

} else {
  print "./echon.pl: argument 1 must be a non-negative integer\n";
  exit 0;
}

File: 9041/lab06/echon.py
#!/usr/bin/python3
import sys


if (len(sys.argv) != 3):
    print ("Usage: ./echon.py <number of lines> <string>")
    sys.exit(0)
if (not(sys.argv[1].isdigit())):
    print ("./echon.py: argument 1 must be a non-negative integer")
    sys.exit(0)
if (int(sys.argv[1]) < 0):
    print ("./echon.py: argument 1 must be a non-negative integer")
    sys.exit(0)



times = sys.argv[1]
content = sys.argv[2]


for i in range (int(times)):
    print (content)
